test/CMakeLists.txt

```
@@ -1382,7 +1382,6 @@ endif()
 # Add here regression tests that should be run only on Vega10/20 and only with FP16.
 add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_DISABLED
```

The workaround comment has been deleted, but what about the test itself? Let's actually enable the test for gfx908:
```suggestion
add_custom_test(test_regression_half_vega FLOAT_DISABLED HALF_ENABLED GFX908_ENABLED
```